model/src/main/kotlin/config/PathExclude.kt

```diff
@@ -46,9 +45,8 @@ data class PathExclude(
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val comment: String = ""
 ) {
-    private val glob by lazy {
-        FileSystems.getDefault().getPathMatcher("glob:${pattern.removePrefix("./")}")
-    }
-
-    fun matches(path: String) = glob.matches(Paths.get(path))
+    fun matches(path: String) = FileMatcher.match(
+        pattern = pattern.removePrefix("./"),
```

Comment:
I have a hunch that `removePrefix("./")` could be dropped now. Could you please verify that?

Reply:
I ran `println(FileMatcher.match("./x", "x"))` which outputs `false`.